api/events.go

```diff
@@ -115,6 +120,17 @@ func (eh *eventsHandler) Handle(e *cluster.Event) error {
 	data = data[:len(data)-1]
 	data = append(data, []byte(node)...)
```

Comment:
Unrelated to the changes in this PR, but this is really weird. Not sure why `node` can't just be a struct field that gets serialized the normal way.